src/components/DownloadView/index.js

```diff
@@ -103,6 +110,21 @@ export default ({ handleInput, onDownload, onClose }) => (
               />
             </td>
           </tr>
+          <tr>
```

Comment:
I would prefer it if this whole row were hidden when `enableResolution === false`, rather than just being disabled.

Reply:
Sure, that's easy enough to change.